app/src/main/java/fr/free/nrw/commons/nearby/NearbyMapFragment.java

```diff
@@ -131,13 +128,24 @@
     private Bundle bundleForUpdates;// Carry information from activity about changed nearby places and current location
     private boolean searchedAroundCurrentLocation = true;
 
-    @Inject @Named("application_preferences") BasicKvStore applicationKvStore;
-    @Inject @Named("defaultKvStore") BasicKvStore prefs;
-    @Inject @Named("direct_nearby_upload_prefs") JsonKvStore directKvStore;
-    @Inject @Named("default_preferences") BasicKvStore defaultKvStore;
-    @Inject BookmarkLocationsDao bookmarkLocationDao;
-    @Inject ContributionController controller;
-    @Inject Gson gson;
+    @Inject
+    @Named("application_preferences")
+    BasicKvStore applicationKvStore;
+    @Inject
+    @Named("defaultKvStore")
+    BasicKvStore prefs;
+    @Inject
+    @Named("direct_nearby_upload_prefs")
+    JsonKvStore directKvStore;
+    @Inject
+    @Named("default_preferences")
+    BasicKvStore defaultKvStore;
+    @Inject
+    BookmarkLocationsDao bookmarkLocationDao;
+    @Inject
+    ContributionController controller;
+    @Inject
+    Gson gson;
```

Comment:
Was this change really necessary?

Reply:
Changes was not done by me when i pulled the repository the changes has been imported